apple/bundling/bundling_support.bzl

```diff
@@ -284,9 +284,6 @@ def _validate_bundle_id(bundle_id):
       for ch in part:
         if ch != "-" and not ch.isalnum():
           fail("Invalid character(s) in bundle_id: \"%s\"" % bundle_id)
-  if len(bundle_id_parts) < 2:
-    fail("bundle_id isn't at least 2 segments: \"%s\"" % bundle_id)
-
 
```

Comment:
please keep the 2 blank lines separation

Reply:
Ah good catch - updated in the latest revision!